### PR TITLE
form update, confirm form or edit, save form, submit form on confirma…

### PR DESCRIPTION
**Pull request by eselkin**
…tion, and complete flow through Position Final and implement close

### CODE REVIEW

File: src/container/FormContainer.jsx [hunk 1]
```diff
@@ -4,117 +4,128 @@ import { connect } from 'react-redux';
 import FormComponent from '../component/FormComponent.jsx';
 import ReactDOM from 'react-dom';
 import ReCAPTCHA from 'react-google-recaptcha';
-import { Button, Icon, Modal } from 'semantic-ui-react';
 import { setHours, setMinutes } from 'date-fns';
+import { verifiedCaptcha, resetForm, saveForm, editForm, submitForm } from '../actions/Form';
+import ConfirmFormContentComponent from '../component/ConfirmFormContentComponent';
+import PositionFormFinalStep from '../component/PositionFormFinalStep';
```
Comment (jvillama) [inline src/container/FormContainer.jsx]: Do ConfirmFormContentComponent and PositionFormFinalStep need the jsx extension also?

File: src/component/AgendaItem.jsx [hunk 2]
```diff
@@ -1,114 +1,99 @@
...
-//Now I just need to design a page and put informations on them
-import { Button, Card, Container, Loader } from "semantic-ui-react";
+import format from 'date-fns/format';
+// I can just get the id from the param and use to to fetch from Application state's agenda agendaitems
+// Now I just need to design a page and put informations on them
+import { Button, Card, Container, Loader } from 'semantic-ui-react';
+
+const recommendationReducer = (acc, curr) => `${acc}<br />${curr}`;
+const summaryReducer = (acc, curr) => {
+  if (!this.gotBackground) {
```
Comment (jvillama) [inline src/component/AgendaItem.jsx]: Receiving "Uncaught TypeError: Cannot read property 'gotBackground' of undefined". Possibly update to `if (this && !this.hasOwnProperty('gotBackground')) {` as well as in AgendaFeedContainer.jsx

Comment (eselkin) [reply src/component/AgendaItem.jsx]: Yeah, that was a stupid mistake... let me correct that (I have to put the function back in the class scope)

**Comment by eselkin** [comment]
update to AgendaItem made

**Pull request merged**